### PR TITLE
bugfix/#2989-update-styles-for-ubs-client

### PR DESCRIPTION
**Pull request by dyadyushko**
update ubs client styles
image

### CODE REVIEW

**Review by rostikdoroshenko — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html [hunk 1]
```diff
@@ -49,7 +50,7 @@
   (scrolled)="onScroll()"
   [scrollWindow]="false"
 >
-  <table *ngIf="!isLoading; else spinner" mat-table [dataSource]="dataSource" matSort class="mat-elevation-z0" cdkDropListGroup>
+  <table *ngIf="!isLoading; else spinner" mat-table [dataSource]="dataSource" matSort class="mat-elevation-z0 table" cdkDropListGroup>
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html:53]: Add "th" headers to this table.

Comment (JimHolman1199) [reply src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html:53]: Add a description to this table.

Comment (JimHolman1199) [reply src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html:53]: https://sonarcloud.io/project/issues?id=ita-social-projects-green-city-client&open=AXroheaMtOsjBe3j_bEV&pullRequest=1004&resolved=false&types=BUG

File: src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.scss [hunk 2]
```diff
@@ -133,58 +147,54 @@ input {
...
-
 .export-excel-btn {
   display: flex;
   justify-content: center;
+  align-items: center;
   width: 290px;
   height: 36px;
   background-color: var(--primary-white) !important;
-  border: 1px solid var(--ubs-primary-light-grey) !important;
+  border: 1px solid #1d2830 !important;
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.scss:157]: all colors should be in variables

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.html [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
+    <img [src]="ubsHeaderIcons.ubsAdminLogo" alt="UBS admin logo" />
   </div>
-  <div class="icons">
-    <img src="assets/admin-header/first-icon-admin.svg" alt="Selected items" />
-    <img src="assets/admin-header/bell-icon-admin.svg" alt="Notifications" />
-    <span>En</span>
-    <img src="assets/img/arrow_down.svg" alt="Change language" />
-    <a href="/">Home</a>
+  <nav>
+    <a class="nav-link" href="#">Про нас</a>
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-header/ubs-header.component.html:6]: is it possible to do ngFor here ?

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.html [hunk 4]
```diff
@@ -1,13 +1,19 @@
...
+    <a class="nav-link" href="#">Правила сортування</a>
+    <a class="nav-link" href="#">Еко-магазин</a>
+    <a class="nav-link" href="#">Green City</a>
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-header/ubs-header.component.html:9]: this link should lead to the main page of green city

File: src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html (hunk 1 above)
Comment (dyadyushko) [reply src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html:53]: this bug fixed by adding role="presentation" in bugfix/#3012 pull request

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.scss [hunk 5]
```diff
@@ -1,33 +1,59 @@
...
+  flex-direction: row;
+  column-gap: 24px;
 }
 
-.icons {
-  grid-area: icons;
-  margin: auto 15px;
-  justify-self: end;
+.nav-link {
+  color: #444e55;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-header/ubs-header.component.scss:23]: use variables for colors

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.scss [hunk 6]
```diff
@@ -1,33 +1,59 @@
...
+  font-size: 14px;
+  text-decoration: underline;
+  text-underline-offset: 1.5px;
+  padding: 0;
 
-  .header-user {
-    color: var(--primary-green);
-    font-family: var(--primary-font);
+  &:hover {
+    color: #5639bb;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-header/ubs-header.component.scss:30]: use variables for colors

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.scss [hunk 7]
```diff
@@ -1,33 +1,59 @@
...
+  align-items: center;
+  column-gap: 15px;
+}
+
+.language-change {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  column-gap: 7.5px;
+  color: #64727d;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-header/ubs-header.component.scss:46]: use variables for colors

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.scss [hunk 8]
```diff
@@ -1,33 +1,59 @@
...
+}
+
+.header-user {
+  color: #5639bb;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-header/ubs-header.component.scss:50]: use variables for colors

File: src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.scss (hunk 2 above)
Comment (dyadyushko) [reply src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.scss]: done

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.html (hunk 3 above)
Comment (dyadyushko) [reply src/app/ubs-admin/components/ubs-header/ubs-header.component.html]: done

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.html (hunk 4 above)
Comment (dyadyushko) [reply src/app/ubs-admin/components/ubs-header/ubs-header.component.html]: done

File: src/app/ubs-admin/components/ubs-header/ubs-header.component.scss (hunk 5 above)
Comment (dyadyushko) [reply src/app/ubs-admin/components/ubs-header/ubs-header.component.scss:23]: done

**Review by JimHolman1199 — APPROVED** [review]

**Review by s1auka — APPROVED** [review]

**Review by YuriShevchuk — APPROVED** [review]

**Comment by JimHolman1199** [comment]
please fix conflicts

**Comment by dyadyushko** [comment]
> please fix conflicts

done

**Pull request merged**